weldx/util.py

```diff
@@ -1335,3 +1336,17 @@ def compare_nested(
 
 
 compare_nested = _Eq_compare_nested.compare_nested
+
+
+def is_interactive_session() -> bool:
```

Comment:
I would be really interested to see if/how we can distinguish between jupyter notebook/lab here

Reply:
It doesn't seem to be possible as both environments seem to be equal from within the ipython object. 
There has been a request to enable the display of JSON, but there was no response since last year. So I guess this has no priority at all. https://github.com/jupyter/notebook/issues/5446